src/App.tsx

```diff
@@ -1,24 +1,134 @@
+import { InputForm } from './InputForm';
+import { Notifications } from './Notifications';
+import { TodoList } from './TodoList';
 import { UserWarning } from './UserWarning';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterTypeEnum';
 
-const USER_ID = 0;
+const USER_ID = 10592;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
-
-  return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">React Todo App - Load Todos</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
-  );
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [removeTodoIsClicked, setRemoveTodoIsClicked] = useState(false);
+  const [editTodoIsClicked, setEditTodoIsClicked] = useState(false);
+  const [filterMode, setFilterMode] = useState<FilterType>(FilterType.All);
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [fetchTodosStatus, setFetchTodosStatus] = useState(false);
+  const [onEmptyFormSubmit, setOnEmptyFormSubmit] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  const allTodosCompleted = useMemo(() => {
+    return filteredTodos.every((todo) => todo.completed);
+  }, [filteredTodos]);
+
+  const notAllTodosCompleted = useMemo(() => {
+    return filteredTodos.every((todo) => todo.completed);
+  }, [filteredTodos]);
+
+  const showNotification = removeTodoIsClicked
+  || editTodoIsClicked
+  || onEmptyFormSubmit;
+
+  const getTodos = async () => {
+    try {
+      const data = await client.get(`/todos?userId=${USER_ID}`) as Todo[];
+
+      setTodos(data);
+      setFetchTodosStatus(true);
+    } catch {
+      setFetchTodosStatus(false);
+    }
+  };
+
+  useEffect(() => {
+    getTodos();
+  }, []);
+
+  const filteredTodosArr = useMemo(() => {
+    return todos.filter((todo) => {
+      if (filterMode === FilterType.Active) {
+        return !todo.completed;
+      }
+
+      if (filterMode === FilterType.Completed) {
+        return todo.completed;
+      }
+
+      return todo;
+    });
+  }, [todos, filterMode]);
+
+  useEffect(() => {
+    setFilteredTodos(filteredTodosArr);
+  }, [filteredTodosArr]);
+
+  return USER_ID
+    ? (
+      <div className="todoapp">
+        <h1 className="todoapp__title">todos</h1>
+
+        <div className="todoapp__content">
+          <header className="todoapp__header">
+            {/* this buttons is active only if there are some active todos */}
+            <button
+              type="button"
+              className={classNames('todoapp__toggle-all', {
+                active: allTodosCompleted,
+                inactive: notAllTodosCompleted,
+              })}
+            />
```

Comment:
The 'todoapp__toggle-all' button doesn't seem to have an onClick event attached to it. If this button is for toggling all todos, you should add a function that handles this.